.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+
+
+#### Brief description of what is fixed or changed
+
+<!-- If this pull request fixes an issue, write "Fixes #NNNN" in that exact
+format, e.g. "Fixes #1234". See
+https://github.com/blog/1506-closing-issues-via-pull-requests . Please also
+write a comment on that issue linking back to this pull request once it is
+open.
+
+OR/AND
+
+Describe your changes here
+-->
+
+#### Deployed URL
+
+<!-- replace <pr_number> with your pr number -->
+
+https://pr-<pr_number>-scipy-sphinx-theme-v2.surge.sh
```

Comment:
Should be `http://pr-<pr_number>-numpy.org.surge.sh/` instead I think? It actually has `-newsite` in the name as well, that would be good to drop.

Reply:
Oh! my bad I will update it.